scripts/upgrade.sh

```diff
@@ -32,14 +32,18 @@ if [ -x "$(command -v awk)" ] && [ -x "$(command -v sed)" ] ; then
 fi
 
 # Backup database
-SQL_BACKUP_PATH="$PEERTUBE_PATH/backup/sql-peertube_prod-$(date +"%Y%m%d-%H%M").bak" 
-DB_USER=$(node -e "console.log(require('js-yaml').safeLoad(fs.readFileSync('$PEERTUBE_PATH/config/production.yaml', 'utf8'))['database']['username'])")
-DB_PASS=$(node -e "console.log(require('js-yaml').safeLoad(fs.readFileSync('$PEERTUBE_PATH/config/production.yaml', 'utf8'))['database']['password'])")
-DB_HOST=$(node -e "console.log(require('js-yaml').safeLoad(fs.readFileSync('$PEERTUBE_PATH/config/production.yaml', 'utf8'))['database']['hostname'])")
-DB_SUFFIX=$(node -e "console.log(require('js-yaml').safeLoad(fs.readFileSync('$PEERTUBE_PATH/config/production.yaml', 'utf8'))['database']['suffix'])")
-mkdir -p $PEERTUBE_PATH/backup
-
-PGPASSWORD=$DB_PASS pg_dump -U $DB_USER -h $DB_HOST -F c "peertube${DB_SUFFIX}" -f "$SQL_BACKUP_PATH"
+if [ "$(whereis pg_dump | cut -d ":" -f 2)" != "" ]
```

Comment:
Using the same than https://github.com/Chocobozzz/PeerTube/blob/develop/scripts/upgrade.sh#L23 is not simpler?